platforms/core-runtime/concurrent/src/main/java/org/gradle/internal/concurrent/CompositeStoppable.java

```diff
@@ -75,11 +68,20 @@ public CompositeStoppable add(Object... elements) {
         return this;
     }
 
-    public synchronized CompositeStoppable add(Object closeable) {
-        this.elements.add(toStoppable(closeable));
+    public CompositeStoppable add(Object closeable) {
+        Stoppable stoppable = toStoppable(closeable);
+        if (null != stoppable) {
```

Comment:
💅 
```suggestion
        if (stoppable != null) {
```

feels more conventional